client/my-sites/sidebar/style.scss

```
@@ -519,9 +519,7 @@ $font-size: rem(14px);
 		}
 
 		.sidebar__menu.is-togglable .sidebar__heading {
-			padding: 12px 10px;
-			margin: 0 12px;
-			border-radius: 4px;
+			padding: 10px;
```

```suggestion
			box-sizing: border-box;
```

Let's keep the existing padding and use `box-sizing` as `.sidebar__menu-link` is already using the `box-sizing` so this change will make the styling consistent.

---

Updated in https://github.com/Automattic/wp-calypso/pull/101069/commits/8d1e2c724383494354436ae9344a12f25fea38bd